napari/layers/_layer_actions.py

```diff
@@ -31,6 +32,13 @@
     from ..utils.context._expressions import Expr
 
 
+@inject_napari_dependencies
+def _show_layer_info(ll: LayerList):
+    layer = ll.selection.active
+    dlg = QtLayerInfoDialog(layer=layer)
+    dlg.exec_()
+
+
```

Comment:
this module should have no Qt dependencies. Please select a proper place in `_qt` module like here https://github.com/napari/napari/blob/1546f18ecc13364d5415623a9c11ed760ff043e2/napari/_qt/menus/file_menu.py#L209-L211